src/Game.h

```diff
@@ -98,6 +98,17 @@ class Game {
 
 	void SetTimeAccel(TimeAccel t);
 	void RequestTimeAccel(TimeAccel t, bool force = false);
+    
```

Comment:
Nitpick of the day: Technically, there's a trailing indentation on this empty line. I think we usually don't indent empty lines.